File: VB/ColorHelper/CellsHightlightHelper.vb
Imports System.Collections.ObjectModel
Imports System.Windows
Imports DevExpress.Xpf.Grid

Namespace WpfApplication
    Public NotInheritable Class CellsHightlightHelper
        Public Shared ReadOnly CellsToHighlightProperty As DependencyProperty = DependencyProperty.RegisterAttached("CellsToHighlight", GetType(ObservableCollection(Of HighlightedGridCell)), GetType(CellsHightlightHelper), Nothing)

        Public Shared Function GetCellsToHighlight(ByVal target As DependencyObject) As ObservableCollection(Of HighlightedGridCell)
            Return CType(target.GetValue(CellsToHighlightProperty), ObservableCollection(Of HighlightedGridCell))
        End Function
        Public Shared Sub SetCellsToHighlight(ByVal target As GridControl, ByVal value As ObservableCollection(Of HighlightedGridCell))
            target.SetValue(CellsToHighlightProperty, value)
        End Sub
    End Class
End Namespace

File: CS/ColorHelper/CellsHightlightHelper.cs
using System.Collections.ObjectModel;
using System.Windows;
using DevExpress.Xpf.Grid;

namespace WpfApplication {
    public static class CellsHightlightHelper {
        public static readonly DependencyProperty CellsToHighlightProperty = DependencyProperty.RegisterAttached("CellsToHighlight", typeof(ObservableCollection<HighlightedGridCell>), typeof(CellsHightlightHelper), null);

        public static ObservableCollection<HighlightedGridCell> GetCellsToHighlight(DependencyObject target) {
            return (ObservableCollection<HighlightedGridCell>)target.GetValue(CellsToHighlightProperty);
        }
        public static void SetCellsToHighlight(GridControl target, ObservableCollection<HighlightedGridCell> value) {
            target.SetValue(CellsToHighlightProperty, value);
        }
    }
}